frontend/src/components/Pages/FrontPage/MissionOverview/StopDialogs.tsx

```diff
@@ -50,11 +52,47 @@ export enum MissionStatusRequest {
     Resume,
 }
 
-export const StopMissionDialog = ({ missionName, robotId }: MissionProps): JSX.Element => {
+export const StopMissionDialog = ({ missionName, robotId, missionTaskType }: MissionProps): JSX.Element => {
     const { TranslateText } = useLanguageContext()
     const [isStopMissionDialogOpen, setIsStopMissionDialogOpen] = useState<boolean>(false)
     const { updateRobotMissionState } = useMissionControlContext()
 
+    const dialogContent = (missionTaskType: TaskType) => {
```

Comment:
```suggestion
    const DialogContent = (missionTaskType: TaskType) => {
```
Generally try to have PascalCase if returning a JSX element and camelCase if not